_posts/2020-09-21-permission-editor.md

```diff
@@ -0,0 +1,110 @@
+
+## Customize it as you want 🔧🔨
+
+sensenet has a lot of built-in permission and we also keep some for your special needs:  Custom01, Custom02, etc. You can also group these permissions for the permission editor dialog as you want. The related settings file is available under Setup menu / Permission.settings.
+
+The default settings are the following:
+
+```{
+   "groups":[
+      {
+         "Read":["See", "Preview", "PreviewWithoutWatermark", "PreviewWithoutRedaction", "Open"]
+      },
+      {
+         "Write":[ "Save", "AddNew", "Delete"]
+      },
+      {
+         "Versioning":["OpenMinor", "Publish", "ForceCheckin", "Approve", "RecallOldVersion"]
+      },
+      {
+         "Advanced":["SeePermissions", "SetPermissions", "RunApplication"]
+      }
+   ]
+}
+```
+
+On the permission editor dialog you can see the groups and permissions based on these settings. 
+
+<p align="center">
+<img src="/img/posts/permission_groups.png">
+</p>
+
+## How to use it? 🎮
+
+Select a content from your repository, right-click on it and select 'Set permissions' from the dropdown. On the permission view you can check the entries which has inherited or set permissions. Selecting an entry makes the permission editor dialog open.
+
+On the dialog turn the switchers to on/off to enable/disable a permission.  On groups there are also switchers as well: you can easily enable/disable all of the permission under the selected group... and there is the king of kings: Full access 👑 (you can turn on/off all permissions)
+
+Some permissions are disabled because they are inherited and you cannot override them. The changes only apply on clicking Submit button so for your comfort there is also a reset button to restore the unsaved changes.
+
+<p align="center">
+<img src="/img/posts/permission_how_to_use.gif">
+</p>
+
+## Additional features 💄👓👜
+
+**Make content public/private**
+
+If you would like to share your content (content can be almost anything, cause in sensenet everything is a content) with somebody e.g. from another company who has no authority to your repository you can do it easily by clicking on "Make content public" button. If you change your mind reverting this is easy as it was to set.
+
+<p align="center">
+<img src="/img/posts/permission_make_public.gif">
+</p>
+
+**Assign new permission**
+
+If you would like to add permission for someone who has no permisison at all on this content you can do it with assign new permission button. Select the user who needs the rights and turn on/off the switchers 😉
+
+<p align="center">
+<img src="/img/posts/permission_assign_new.gif">
+</p>
+
+**Local only**
+
+There are cases however when you do not want child content to inherit a permission entry. For example you want to allow certain users to see a content (e.g. a Content List) but you do not want them to be able to see content that were added to that list. You can mark the permission entry as 'local-only' to avoid permission inheritence.
```

Comment:
```suggestion
One of the big advantages of sensenet content repository tree is inheritance that is applied on permissions as well. However, there are cases when you do not want child contents to inherit permissions. For example you want to allow certain users to see a content (e.g. a Content List) but you do not want them to be able to see contents that are inside this container. To avoid permission inheritance, mark permission entries as 'local-only'.
```